src/common/resources.rs

```diff
@@ -39,22 +40,30 @@ impl CpuRequest {
     }
 }
 
-#[derive(Serialize, Deserialize, Debug, Clone, Hash, Eq, PartialEq)]
+pub type TimeRequest = Duration;
+
+#[derive(Default, Serialize, Deserialize, Debug, Clone, Hash, Eq, PartialEq)]
 pub struct ResourceRequest {
     cpus: CpuRequest,
+
+    #[serde(default)]
+    time: TimeRequest, // Minimal remaining time of the worker life time needed to START the task
```

Comment:
`time` -> `min_required_time`? Or some other name, `time` is too general.

Reply:
It is in a struct Request so i did not wanted repeat that, but min_time is probably better than just time